shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -261,5 +261,85 @@ TEST_F(EmbedderTest, IsolateServiceIdSent) {
   kill_latch.Wait();
 }
 
+TEST_F(EmbedderTest, ShouldBeAbleToCreateAndCollectCallbacks) {
```

Comment:
Suggestion for this and future tests: We should have a brief comment explaining what the test does.

Reply:
Done.